lib/asciidoctor/converter/semantic-html5.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+require 'date'
```

Comment:
I tend to prefer autoload. That way, it's loaded on demand.

```
autoload :Date, 'date'
```

Reply:
I should probably hide it from Opal using:

```rb
unless RUBY_ENGINE_OPAL
  autoload :Date, 'date'
end
```

Not sure if it's working properly with the following syntax:

```rb
autoload :Date, 'date' unless RUBY_ENGINE_OPAL
```

Reply:
Yes. You can see how I do it here: https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor.rb#L5-L15